src/Orckestra.Composer.Website/UI.Package/Sass/Custom/Search/_facets.scss

```diff
@@ -32,3 +107,99 @@
     }
 }
 
+.facets-modal {
+
+    .facets-modal-header, 
+    .facets-modal-footer  {
+        display: none;
+    }
+
+    @include media-breakpoint-down(md) { 
+        display: none;
+        position: fixed;
+        top: 0;
+        left: 0;
+        bottom: 0;
+        right: 0;
+        z-index: 1031;
+        background: #fff;
+        outline:0;
+        width: 100%;
+        height: 100%;
+        z-index: 1041;
+        &.show {
+            display: block;
+        }
+
+        .facets-modal-dialog {
+            position: relative;
+            pointer-events: none;
+            width: auto;
+            flex-direction: column;
+            height: 100%;
+            display: flex;
+            max-height: subtract(100%, $modal-dialog-margin * 2);
+        }
+
+        .facets-modal-content {
+            position: relative;
+            display: flex;
+            flex-direction: column;
+            overflow: hidden;
+            width: 100%;
+            pointer-events: auto;
+            max-height: subtract(100vh, $modal-dialog-margin * 2);
+        }
+
+        .facets-modal-header {
+            flex-shrink: 0;
+            display: flex;
+            align-items: flex-start;
+            justify-content: space-between;
+            padding: 10px;
+
+            .close {
+                padding: 20px;
+            }
+        }
+
+        .facets-modal-body {
+            position: relative;
+            overflow-y: auto;
+            flex: 1 1 auto;
+           //TODO min-height: subtract(100vh, 120px);
```

Comment:
Is it finished?(TODO:)

Reply:
no, but I added more information into TODO to invetigate later. It doesn't work just on my iPhone xS, but looks like works on others phones.